ansible/roles/network/tasks/main.yml

```diff
@@ -50,3 +50,9 @@
     minute: 0
     job: /usr/bin/python /usr/sbin/screenly_net_watchdog.py
     state: absent
+
+- name: ethN names for interfaces
+  command: "{{item}}"
```

Comment:
Check out this:
https://github.com/Screenly/screenly-ose/blob/master/ansible/roles/system/tasks/main.yml#L36-L41

We already have some logic for dealing with `cmdline.txt`, so let's reuse that.

Reply:
hmm. but in this code there is handling for berryboot. or do you mean just place a new task for cmdline.txt in the same file?

Reply:
No, check the lines before. It's the opposite. It's *only* triggered it's *not* BerryBoot.